leaderboard/web/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -6,11 +6,26 @@ import 'rxjs/add/observable/interval';
 import 'rxjs/add/operator/map';
 
 interface TeamResponse {
+  teamName: string;
+  downTimeSeconds: number;
+  points: number;
+}
+
+interface Team {
   name: string;
   uptime: number;
   uppercent: number;
   point: number;
 }
+const TOTAL_TIME = 68400;
+function convertTeam(teamResponse: TeamResponse): Team {
+   return {
+    'name': teamResponse.teamName,
+    'uptime': (TOTAL_TIME - teamResponse.downTimeSeconds),
+    'uppercent': ((TOTAL_TIME - teamResponse.downTimeSeconds) / TOTAL_TIME) * 100,

```

Comment:
Per our thread, we want to get away from the % of uptime and use current uptime status (Red/Green) instead.  We should just display the total downTimeSeconds in minutes.  i.e. 30 seconds downtime = .5 minutes

Reply:
Thank you for the review. I understand the change. I know the change happens. However, why I delete ‘not merge’ is I wanted to keep this repo working. I took step-by-step.  

1. Fix the issue 
2. Introduce a change

Kent Beck said that we should not ware two hats at the same time. I just follow the princial.  However, if you like I’d like to introduce the change to this branch. It might take time to investigate the new UI parts.

Reply:
In general, I agree with Kent Beck :).  In this case, we have a demo environment and not sure those rules apply as much.  At any rate, no worries - I will go ahead and approve.

Reply:
Thanks. I'll create a new branch to implement the new UI. :)